### PR TITLE
Automate test_positive_ansible_variables_installed_with_collection

### PR DESCRIPTION
**Pull request by Gauravtalreja1**
### Problem Statement

1. test_positive_ansible_variables_installed_with_collection is stubbed/Manual test.

### Solution
1. Automate test_positive_ansible_variables_imported_with_roles for admin and non-admin users
Remove unrelated tests

### Related Issues
This PR includes changes from https://github.com/SatelliteQE/robottelo/pull/16431, this needs to be rebased after merge

### CODE REVIEW

**Comment by Gauravtalreja1** [comment]
trigger: test-robottelo
pytest: tests/foreman/cli/test_ansible.py::TestAnsibleCfgMgmt::test_positive_ansible_variables_installed_with_collection

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 8794
Build Status: SUCCESS
PRT Comment: pytest tests/foreman/cli/test_ansible.py::TestAnsibleCfgMgmt::test_positive_ansible_variables_installed_with_collection --external-logging
Test Result : ================== 1 passed, 9 warnings in 686.25s (0:11:26) ===================
```

File: tests/foreman/cli/test_ansible.py [hunk 1]
```diff
@@ -186,6 +191,61 @@ def test_add_and_remove_ansible_role_hostgroup(self, target_sat):
...
+    @pytest.mark.rhel_ver_list([settings.content_host.default_rhel_version])
+    def test_positive_ansible_variables_installed_with_collection(
+        self, request, target_sat, module_org, module_ak_with_cv, rhel_contenthost
+    ):
+        """Verify that installing an Ansible collection also imports
+        any variables associated with the roles avaialble in the collection
+
+        :id: 7ff88022-fe9b-482f-a6bb-3922036a1e1c
+
+        :steps:
```
Comment (nofaralfasi) [inline tests/foreman/cli/test_ansible.py:204]: There is no mention of installing the collection here.

Comment (Gauravtalreja1) [reply tests/foreman/cli/test_ansible.py:204]: @nofaralfasi ah, I missed it earlier, updated. Thank you :)

**Comment by Gauravtalreja1** [comment]
trigger: test-robottelo
pytest: tests/foreman/cli/test_ansible.py::TestAnsibleCfgMgmt::test_positive_ansible_variables_installed_with_collection

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 8923
Build Status: SUCCESS
PRT Comment: pytest tests/foreman/cli/test_ansible.py::TestAnsibleCfgMgmt::test_positive_ansible_variables_installed_with_collection --external-logging
Test Result : ================== 1 passed, 9 warnings in 623.92s (0:10:23) ===================
```

**Pull request merged**